### PR TITLE
fix: Replace Ionic assets (icons, splash)

### PR DESCRIPTION
**Pull request by sstur**
## What does this PR accomplish?
This solves #133. Replace all default Ionic assets (icons, splash) with WHO branded assets.

The icons and splash screen consist simply of the blue logo taken from the WHO website:
https://www.who.int/ResourcePackages/WHO/assets/dist/images/logos/en/v-logo-blue.svg

## Did you add any dependencies?
No

## How did you test the change?
Opened in Xcode and Android Studio.

Screen Shot 2020-03-22 at 15 52 30
Screen Shot 2020-03-22 at 15 56 16
Screen Shot 2020-03-22 at 15 57 08

### CODE REVIEW

File: client/ionic/android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@color/ic_launcher_background"/>
-    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
+    <foreground android:drawable="@mipmap/ic_launcher"/>
```
Comment (hspinks) [inline client/ionic/android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: forgive my android ignorance here, but what's the reason for the change here and `ic_launcher_round.xml`?

Comment (sstur) [reply client/ionic/android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: I'm not 100% sure, but my understanding is that Android offers the option to ship both a square oldschool icon and the (optional) modern/rounded icon which is actually two layers, the forground can be masked and hovers over a background (in a way to create a cool effect) but in our case we have just the normal icon, without any foreground/background stuff so I just called it ic_launcher, which I _think_ is the convention. But if someone else knows this Android stuff better please do add some thoughts here.

Comment (sstur) [reply client/ionic/android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: Oh, the short answer to your question is: if I didn't rename this the build fails.

**Comment by SamMousa** [comment]
Merged based on @hspinks LGTM and the explanation for his comment.

**Comment by SamMousa** [comment]
@sstur I have removed this (merge) commit from master on request from @advayDev1.
Please create a new PR make sure to assign it to him.

His comment from Slack:
>  he needs to put those WHO assets in `identity/`

**Comment by advayDev1** [comment]
Thanks Sam!  Simon please put the actual files in identity/ and symlink them wherever you need them.  (See [LICENSE](LICENSE))

**Comment by sstur** [comment]
> Simon please put the actual files in identity/ and symlink them wherever you need them.

OK, will do!

**Comment by advayDev1** [comment]
@sstur what's the status of revamping this PR?
